packages/react-tinacms-github/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "react-tinacms-github",
+  "version": "1.0.0",
```

This will get bumped automatically when the package is released
```suggestion
  "version": "0.0.0",
```